app/move/controllers/unassign.js

```diff
@@ -0,0 +1,64 @@
+const FormWizardController = require('../../../common/controllers/form-wizard')
+const allocationService = require('../../../common/services/allocation')
+const moveService = require('../../../common/services/move')
+
+class UnassignController extends FormWizardController {
+  middlewareChecks() {
+    this.use(this.checkAllocation)
+    super.middlewareChecks()
+  }
+
+  async checkAllocation(req, res, next) {
```

Comment:
Sorry, missed this last time but you can drop the `async` as you're not doing an `await` anymore.

Reply:
👍